tests/platform_tests/daemon/test_chassisd.py

```diff
@@ -0,0 +1,213 @@
+"""
+Check daemon status inside PMON container. Each daemon status is checked under the conditions below in this script:
+* Daemon Running Status
+* Daemon Stop status
+* Daemon Restart status
+
+This script is to cover the test case in the SONiC platform daemon and service test plan:
+https://github.com/Azure/sonic-mgmt/blob/master/docs/testplan/PMON-Services-Daemons-test-plan.md
+"""
+import logging
+import re
+import time
+
+from datetime import datetime
+
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.daemon_utils import check_pmon_daemon_enable_status
+from tests.common.platform.processes_utils import wait_critical_processes, check_critical_processes
+from tests.common.utilities import compose_dict_from_cli, skip_release, wait_until
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t2'),
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer
+]
+
+expected_running_status = "RUNNING"
+expected_stopped_status = "STOPPED"
+expected_exited_status = "EXITED"
+
+daemon_name = "chassisd"
+
+SIG_STOP_SERVICE = None
+SIG_TERM = "-15"
+SIG_KILL = "-9"
+
+STATE_DB = 6
+
+@pytest.fixture(scope="module", autouse=True)
+def setup(duthosts, rand_one_dut_hostname):
```

Comment:
Instead of rand_one_dut_hostname, can we use one of the enum_rand_one fixtures.

Reply:
sure will do, thanks - I am yet to validate this test case.